avocado/utils/memory.py

```diff
@@ -48,6 +48,13 @@ def memtotal():
     return read_from_meminfo('MemTotal')
 
 
+def availablememtotal():
+    """
+    Read ``MemAvailable`` from meminfo.
+    """
+    return read_from_meminfo('MemAvailable')
```

Comment:
Actually I'd even skip the docstring improvement. People can google-search for it...